pom.xml

```diff
@@ -5,22 +5,31 @@
 	<parent>
 		<groupId>org.springframework.boot</groupId>
 		<artifactId>spring-boot-starter-parent</artifactId>
-		<version>2.6.3</version>
+		<version>2.2.5.RELEASE</version>
 		<relativePath/> <!-- lookup parent from repository -->
 	</parent>
 	<groupId>it.pagopa.pm.gateway</groupId>
 	<artifactId>pagopa-payment-transactions-gateway</artifactId>
-	<version>0.0.1-SNAPSHOT</version>
+	<version>${revision}</version>
 	<packaging>war</packaging>
 	<name>pagopa-payment-transactions-gateway</name>
 	<description>Payment transactions gateway for the pagoPA - Payment Manager</description>
 	<properties>
-		<java.version>11</java.version>
+		<java.version>1.8</java.version>
+		<revision>0.0.0</revision>
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+		<project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 	</properties>
 	<dependencies>
 		<dependency>
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-web</artifactId>
+			<exclusions>
+				<exclusion>
```

Comment:
good for deploy on Jboss, but how can we run it locally? maybe with a dedicated maven profile?